src/library.js

```diff
@@ -476,7 +476,7 @@ LibraryManager.library = {
     return {{{ DYNAMICTOP_PTR }}};
   },
 
-#if ABORTING_MALLOC
+#if ABORTING_MALLOC && !ALLOW_MEMORY_GROWTH
```

Comment:
Note that this one is not technically required, as the lower one will not link it in, unless I'm missing something. But it may help with readability, to clarify when it is needed, so looks good!